src/types.js

```diff
@@ -553,7 +553,7 @@ export type UnreadStream = {
 export type NotificationCommon = {
   alert: string,
   content: string,
-  content_truncated: string, // boolean
+  content_truncated: boolean,
```

Comment:
Are you sure this is correct?
I remember declaring it like that few months ago, because the incoming data was `string`, but we wanted it boolean.

Reply:
I did a little digging before this change. I checked where `NotificationCommon` is being referred at and I found it's referenced in `NotificationPrivate` which is again referenced in `Notification`. Then I found that `Notification` is referred to in `config.js` to define the type of Config. Below (line 32), in config,js `undefined` was assigned to `notification`.
I headed on over to `notification.js` where the `notification` was assigned with a value `notificationData.getData()` and `notificationData` is defined as an `Object` whose contents were unknown to me. The TravisCI test showed no errors and i assumed that it was correct. I am not 100% sure that incoming data is actually `boolean`.

Reply:
That would be a mistake on the Zulip's server payload, but I believe we didn't fix that. @jackrzhang might know something about that?

Reply:
The server code sure handles this field as a boolean: search for `content_truncated`, and it comes from `truncate_content` which returns a boolean.

There are several layers between that Python code in the Zulip server and this JS code, though. It wouldn't be a big surprise if one of them turns, say, the JSON value `true` into a string `'true'`.

Note that this is a field we never reference in our code: grep for `content_truncated`. So (a) running tests doesn't give much information about whether an annotation here is right, and (b) this isn't one of the higher-priority things for us to nail down the correct type for.

I've just gone and updated #2052 with current stats and a bit of guidance on finding the highest-value areas to add types now:
> #2971 is a great source of high-value places to add types. Running `yarn run test:flow-coverage` remains another great source.

Want to try those?